src/main/java/com/github/vkorobkov/jfixtures/domain/Row.java

```diff
@@ -28,4 +29,24 @@ public Row columns(Map<String, ?> toMerge) {
         );
         return new Row(name, merged);
     }
+
+    public Row columns(Object... keyValuePairs) {
+        int pairsLength = keyValuePairs.length;
+
+        if (pairsLength % 2 != 0) {
+            throw new IllegalArgumentException("Odd number of key/value pairs");
+        }
+
+        Map<String, Object> keyValueMap = new LinkedHashMap<>();
```

Comment:
A performance trick: I guess we can set the size of `LinkedHashMap` since we know it:
```java
Map<String, Object> keyValueMap = new LinkedHashMap<>(keyValuePairs.length / 2);
```